WebService/ComparisonEngineUI/ComparisonEngineUI/Views/DeleteDrinkPage.xaml.cs

```diff
@@ -5,21 +5,82 @@
 using System.Threading.Tasks;
 using Xamarin.Forms;
 using Xamarin.Forms.Xaml;
+using ComparisonEngineUI.Models;
 using ComparisonEngineUI.ViewModels;
+using ComparisonEngineUI.Data;
 
 namespace ComparisonEngineUI.Views
 {
     [XamlCompilation(XamlCompilationOptions.Compile)]
     public partial class DeleteDrinkPage : ContentPage
     {
+        private List<Bar> _barList { get; set; }
+        public List<Bar> BarList
+        {
+            get
+            {
+                return _barList;
+            }
+            set
+            {
+                if (value != _barList)
+                {
+                    _barList = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
+        private List<Drink> _drinkList { get; set; }
+        public List<Drink> DrinkList
+        {
+            get
+            {
+                return _drinkList;
+            }
+            set
+            {
+                if (value != _drinkList)
+                {
+                    _drinkList = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
+        private List<Drink> initialDrinkList { get; set; }
+        private List<Bar> initialBarList { get; set; }
         public DeleteDrinkPage()
         {
             InitializeComponent();
             this.BindingContext = new DeleteDrinkViewModel();
+            var restService = new RestService();
+            initialBarList = Task.Run(async () => await restService.GetData<List<Bar>>(Constants.BarsUrl)).Result;
+            initialDrinkList = Task.Run(async () => await restService.GetData<List<Drink>>(Constants.DrinksUrl)).Result;
```

Comment:
Is this necessary? We have listContainer.

Reply:
Well I don't think it would be that much effective, however the code would look more readable, but we wrote a lot this way, so changing 2 lines, doesn't really help, since there are a lot more than just two lines.